packages/extension-driver-duckdb/src/lib/sqlBuilder.ts

```diff
@@ -26,15 +26,23 @@ export const isNoOP = (
   return true;
 };
 
+const duckdbExecuteChunkSize =
+  process.env['DUCKDB_EXECUTE_CHUNK_SIZE'] || '2000';
```

Comment:
Suggest to update to mentioned the env in README, and what influence will be if set the `env` to large value or small value.